ext/mysqli/tests/mysqli_connect.phpt

```diff
@@ -118,28 +118,26 @@ require_once('skipifconnectfailure.inc');
         mysqli_close($link);
     }
 
-    if ($IS_MYSQLND) {
-        ini_set('mysqli.default_host', 'p:' . $host);
-            if (!is_object($link = mysqli_connect())) {
-                printf("[021] Usage of mysqli.default_host (persistent) failed\n") ;
-        } else {
-            if (!$res = mysqli_query($link, "SELECT 'mysqli.default_host (persistent)' AS 'testing'"))
-                printf("[022] [%d] %s\n", mysqli_errno($link), mysqli_error($link));
-            $tmp = mysqli_fetch_assoc($res);
-            if ($tmp['testing'] !== 'mysqli.default_host (persistent)') {
-                printf("[023] Result looks strange - check manually, [%d] %s\n",
-                    mysqli_errno($link), mysqli_error($link));
-                var_dump($tmp);
-            }
-            mysqli_free_result($res);
-            mysqli_close($link);
+    ini_set('mysqli.default_host', 'p:' . $host);
+        if (!is_object($link = mysqli_connect())) {
```

Comment:
```suggestion
    if (!is_object($link = mysqli_connect())) {
```
Fix indent please?

Reply:
Thank you! Done as 455e5ca2b9736b918747297507bf2f5819a10c6c.